setup.cfg

```diff
@@ -1,11 +1,19 @@
 [isort]
-line_length = 120
+indent='    '
+line_length=120
+multi_line_output=3
+known_standard_library=mock,six,time,responses
```

Comment:
Here's the source of the odd sorting; of these, only `time` is actually a standard library package under Python 2 (and two of them aren't under Python 3 either).